.coderabbit.yaml

```
@@ -0,0 +1,95 @@
+  max_files: 50
+  
+  # 단일 파일의 최대 라인 수
+  max_lines_per_file: 1000
+
+# 언어별 설정
+language:
+  typescript:
+    # TypeScript 관련 리뷰 옵션
+    strict_mode: true
+    check_types: true
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)